### PR TITLE
Feature/remove muscat diffusers

### PR DESCRIPTION
**Pull request by jnation3406**
I attempted to convert to narrowband_* from diffuser_* and actually hooked in those values, but it seemed like these weren't really used before in the project anyway...

### CODE REVIEW

File: calibrations/facilities/photometric_standards_facility.py [hunk 1]
```diff
@@ -202,6 +200,9 @@ def _build_instrument_config(self):
                 "offset_dec": 0,
                 "defocus": 0
             }
+            base_narrowband_position = None
+            if 'narrobands' in self.cleaned_data and self.cleaned_data['narrowbands']:
```
Comment (phycodurus) [inline calibrations/facilities/photometric_standards_facility.py:204]: is this ('narrobands' vs. 'narrowbands') intentional?

Comment (jnation3406) [reply calibrations/facilities/photometric_standards_facility.py:204]: yeah it was "diffusers" before, so I kept it plural. It seemed to be a setting to set all diffusers/narrowbands to that value rather then setting them each individually.

Comment (markBowman) [reply calibrations/facilities/photometric_standards_facility.py:204]: but it is missing the 'w'

Comment (jnation3406) [reply calibrations/facilities/photometric_standards_facility.py:204]: touche, that's why we do code reviews :)

**Review by phycodurus — APPROVED** [review]

**Comment by phycodurus** [comment]
I've temporarily reverted these changes in PR #62

**Pull request merged**